src/main/java/mat/dto/MeasureTransferDTO.java

```diff
@@ -0,0 +1,17 @@
+package mat.dto;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import mat.client.measure.ManageMeasureDetailModel;
+
+import java.io.Serializable;
+
+@Data
+@NoArgsConstructor
+public class MeasureTransferDTO implements Serializable {
+    private String emailId;
+    private String harpId;
+    private ManageMeasureDetailModel manageMeasureDetailModel;
+    private String measureResourceJson;
+    private String libraryResourcesJson;
```

Comment:
```suggestion
    private String measureResourceElmJson;
    private String libraryResourcesElmJson;
```
The JSON should be the ELM.  Recommend renaming the member variable as above to add some clarity.

Reply:
oh wow, I can commit the change right here. I'll update it. Thanks Joe!